Example/Example/ControlsViewController.swift

```diff
@@ -22,21 +22,25 @@ class ControlsViewController: UIViewController {
   private var subscriptions = Set<AnyCancellable>()
 
   override func viewDidLoad() {
+
     super.viewDidLoad()
 
     Just("Debug Output:")
-      .merge(with: segmented.selectedSegmentIndexPublisher.map { "Segmented at index \($0)" },
+        .merge(with: segmented.selectedSegmentIndexPublisher.map { "Segmented at index \($0)" },
```

Comment:
extra unneeded space

Reply:
I think in general you changed everything to 4-space indents, this project uses two-space indents. Can you please revert that?